ha-sip/src/main.py

```diff
@@ -99,7 +105,25 @@ def main():
         if account_config.enabled:
             sip_accounts[key] = account.create_account(end_point, account_config, command_handler, ha_config, is_first_enabled_account)
             is_first_enabled_account = False
+
+    if mqtt_mode:
+        broker_address = os.environ.get('BROKER_ADDRESS','')
+        port = utils.convert_to_int(os.environ.get('BROKER_PORT','1833'))
+        mqtt_username = os.environ.get('BROKER_USERNAME','')
+        mqtt_password = os.environ.get('BROKER_PASSWORD','')
+        topic = os.environ.get('MQTT_TOPIC','hasip/execute')
+        mqtt_client = MQTTClient(broker_address, port, mqtt_username, mqtt_password, topic, command_handler)
+        mqtt_client.connect()
+
     while True:
+        if mqtt_mode:
+            if not mqtt_client.is_connected():
+                try:
+                    mqtt_client.reconnect()
+                except:
+                    log(None, 'Reconnect to mqtt broker failed. Trying again....')
+                    time.sleep(1)
+            mqtt_client.loop()
```

Comment:
Maybe another thought: can we move this reconnect logic to mqtt_client.loop() and rename loop to handle? This would clean up these internals from main.py